cypress/integration/login.js

```diff
@@ -1,27 +1,35 @@
+import LoginPage from '../objects/LoginPage';
+import TimeLoggin from '../objects/TimeLoggin';
+import Common from '../objects/Common';
+const common = new Common();
+const timeLoggin = new TimeLoggin();
+const loginPage = new LoginPage();
+
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {

```

Comment:
this is a duplicate test case of T2.js
it('Should display validation for empty user after attempted loggin'